packages/calypso-products/src/constants/enterprise.ts

```diff
@@ -0,0 +1,30 @@
+import i18n from 'i18n-calypso';
+
+export const PLAN_ENTERPRISE_FEATURE_LIST_TITLE = i18n.translate(
+	'High performance platform, with:'
+);
+
+export const PLAN_ENTERPRISE_FEATURE_LIST = [
```

Comment:
I'm trying to think whether these features should instead be defined in the regular way, as done for everything else. So this becoming a [FeatureGroup](https://github.com/Automattic/wp-calypso/blob/trunk/packages/calypso-products/src/feature-group-plan-map.ts) that we pass along with the rest, then only Enterprise plan references any of the enclosed features. It could be a `FeatureGroupHighPerformancePlatform`...

Right now, the whole thing feels like a diversion in the logic and data flow.

I would also similarly consider including a "partners-reference" feature and a "partners-reference-feature-group" (or something of this nature), that again, only Enterprise references. This way we won't even need to diverge the UI logic at all for the features-list 🤔

Reply:
Thanks for chiming in.

I moved those outside of the package for now (af585de4c26f9234d4180ae53cc1b38b4187152b). Felt more right given that those features were purely presentational.

Reply:
I can do a follow-up from here too. I think it's possible to have a single form and layering of features & groups, close to what I mentioned. The UI should just fit these in place without additional tweaks/conditioning. We'd also have a cleaner UI without the enterprise plan exclusions. It's actually meant to work this way since the trail-map experiments. 

I'll follow-up on this :-)